pasta/base/annotate.py

```diff
@@ -1490,11 +1490,15 @@ def check_slice_includes_step(self, node):
     def visit_FormattedValue(self, node):
       self.visit(node.value)
       if node.conversion != -1:
-        self.attr(
-            node,
-            'conversion', [self.ws, '!', chr(node.conversion)],
-            deps=('conversion',),
-            default='!%c' % node.conversion)
+        # If we have a self-documenting token, eat the '='.
+        if self.tokens.peek_non_whitespace().src == '=':
```

Comment:
Use instead

`self.optional_token(node, 'self_documenting', '=')`

Reply:
I was looking for that, thanks!